src/README.md

```diff
@@ -13,6 +13,8 @@ pip install -r requirements.txt
 Under src, the following commands help you start the server of the website.
 ```
 export FLASK_APP=surfing_penguin.py
+(export ENV == "Staging" or "Production")
+(export DATABASE_URL= "Your local database url" )
 flask run
 ```
```

Comment:
For deployment on Heroku, please add the process in Readme as well. Also, where and how do you set the environment variables on Heroku? If you are doing this manually, please take a look at this post: https://medium.com/taqtilebr/managing-herokus-app-environment-variables-d13fd99610b